WhateverGreen/kern_igfx.hpp

```diff
@@ -342,7 +429,7 @@ class IGFX {
 	 *  IGHardwareGuC::loadGuCBinary wrapper to feed updated (compatible GuC)
 	 */
 	static bool wrapLoadGuCBinary(void *that, bool flag);
-
+	
```

Comment:
Do not want to be picky, but I get a feeling this is some editor artefact.
Also, I believe 2 spaces are used between * and doc description instead of TABs, e.g.:
`*  Original AppleIntelFramebufferController::ReadRegister32 function`.